pyanaconda/modules/payloads/payload/dnf/tree_info.py

```diff
@@ -158,6 +158,7 @@ def _load_tree_info(self, root_path, file_path=None, file_content=None):
                 tree_info.load(file_path)
 
             tree_info.validate()
+            log.debug("Loaded treeinfo metadata:\n%s", tree_info.dumps())
```

Comment:
I'm not sure if we want to dump the whole treeinfo. Could be a bit more information and maybe something what user don't want to have as part of the logs?

Reply:
We already dump other configurations (like DNF conf). It is hard to guess if Anaconda behaves correctly when you have no idea what's in the treeinfo file. I was really surprised that most of our installations actually work with treeinfo repositories, because there is always the base one.

Reply:
What do you mean by "the base one"? Like the base repository as we request to have it in the Anaconda?

Reply:
Yes, the Anaconda base repository usually has a treeinfo that defines that repository. See: http://cobra02/trees/rawhide/.treeinfo